content/momentum/4/4-tls-macros.md

```diff
@@ -15,71 +15,18 @@ The tls_macros module must be loaded in a configuration file, as follows:
 `tls_macros {}`
 ### <a name="tls_macros.macros"></a> TLS Logging Macros
 
-[Table 71.2, “TLS Logging Macros”](/momentum/4/4-tls-macros#tls-macros-table) lists the TLS logging macros.
-
```

Comment:
This link went to the next line.  I'm guessing it's a holdover from when the below content was elsewere.